app/models/meter.rb

```diff
@@ -33,20 +38,32 @@ class Meter < ApplicationRecord
   scope :inactive, -> { where(active: false) }
   scope :no_amr_validated_readings, -> { left_outer_joins(:amr_validated_readings).where(amr_validated_readings: { meter_id: nil }) }
 
-  enum meter_type: [:electricity, :gas]
+
+  CREATABLE_METERS = [:electricity, :gas].freeze
+  PSEUDO_METERS = [:solar_pv, :exported_solar_pv, :electricity].freeze
+  METER_TYPES = (CREATABLE_METERS + PSEUDO_METERS).uniq
+
+  enum meter_type: METER_TYPES
 
   delegate :area_name, to: :school
 
   validates_presence_of :school, :mpan_mprn, :meter_type
   validates_uniqueness_of :mpan_mprn
 
-  validates_format_of :mpan_mprn, with: /\A[1-3]\d{12}\Z/, if: :electricity?, message: 'for electricity meters should be a 13 digit number'
+  validates_format_of :mpan_mprn, with: /\A[1-3]\d{12}\Z/, if: :traditional_mpan?, message: 'for electricity meters should be a 13 digit number'
+  validates_format_of :mpan_mprn, with: /\A[6,7,9]\d{13}\Z/, if: :pseudo_mpan?, message: 'for electricity meters should be a 13 digit number'
   validates_format_of :mpan_mprn, with: /\A\d{1,10}\Z/, if: :gas?, message: 'for gas meters should be a 1-10 digit number'
 
   def fuel_type
     meter_type.to_sym
   end
 
+  def self.non_gas_meter_types
```

Comment:
Previously we validated and aggregated gas or electricity, we now need to aggregate all of the non-gas meter types